### PR TITLE
Remove quotes from properties

### PR DESCRIPTION
**Pull request by vanillajonathan**
Add missing semicolons.
Use const instead of let.

### CODE REVIEW

File: colors/build.js [hunk 1]
```diff
@@ -16,53 +16,53 @@ function createColor(color, element, formatter) {
 }
 
 const formats = {
-  'css': {
```
Comment (sammaeldaemon) [inline colors/build.js]: css: (

**Comment by bwinton** [comment]
I guess my main question is "Why?"

I kinda understand using const instead of let and adding the missing semicolons help, but I don't see how removing quotes makes the code any more readable, and I'm not a fan of bareword properties when using objects as data, so I'm leaning towards not merging this…

**Comment by andymikulski** [comment]
The issues corrected in this PR are pretty much just linting issues - missing semicolons, `let` vs `const`, etc. Perhaps it'd be more beneficial to add an actual `eslint` script which would catch these issues before PR's are opened? That would create a consistent code style across the repo, as well as provide a concrete reasoning to correct nits like these.

**Comment by bwinton** [comment]
Yeah, that seems reasonable. I'll figure out what the style should be, and add one of those when I can.

In the meantime, let's close this.

**Comment by andymikulski** [comment]
>I'll figure out what the style should be, and add one of those when I can.

For what it's worth, the `eslint --init` command offers a handful of ways to define the style, which will probably be helpful:
```
$ eslint --init
  ? How would you like to configure ESLint? (Use arrow keys)
  ❯ Answer questions about your style
    Use a popular style guide
    Inspect your JavaScript file(s)
```

**Pull request closed without merging**